.github/workflows/docker.yml

```diff
@@ -36,6 +36,6 @@ jobs:
       - name: Deploy the tagged Docker image to Docker Hub
         if: startsWith(github.ref, 'refs/tags/')
         run: |
-          docker login docker.com -u brewtestbot -p ${{secrets.DOCKER_TOKEN}}
```

Comment:
`The push refers to repository [docker.io/homebrew/ubuntu20.04]`
https://github.com/Homebrew/brew/runs/662451638?check_suite_focus=true#step:7:14
Looks like `docker.com` should have been `docker.io` ?